actors/builtin/miner/miner_test.go

```diff
@@ -889,8 +890,6 @@ func TestCommitments(t *testing.T) {
 		// Check new pre-commit in state
 		assert.True(t, upgrade1.Info.ReplaceCapacity)
 		assert.Equal(t, upgradeParams1.ReplaceSectorNumber, upgrade1.Info.ReplaceSectorNumber)
-		// Require new sector's pledge to be at least that of the old sector.
-		assert.Equal(t, oldSector.InitialPledge, upgrade1.PreCommitDeposit)
```

Comment:
Blocking: we're losing important coverage here. The old check isn't right any more, but can you `assert.Equal(t, oldSector.InitialPledge, upgrade1.InitialPledge)` after the prove-commit? Same for upgrade2.

Code earlier on is explicitly manipulating the reward to ensure this property.

Edit: I see code in the `confirmSectorProofsValid` wrapper setting the expectation, but still think we should assert this concrete expectation here. We know exactly what pledge to expect.